src/controllers/paymentsV3Neo.ts

```diff
@@ -0,0 +1,193 @@
+import { NextFunction, Request, Response } from 'express';
+import {
+  Beneficiary,
+  CreatePaymentRequest,
+  CreatePaymentRequestResponse,
+  ProviderSelection,
+  SchemaSelection
+} from 'models/v3/payments-api/create_payment';
+import AuthenticationClient from 'clients/authentication-client';
+import PaymentsClient from 'clients/paymentv3-client';
+import config from 'config';
+import { HttpException } from 'middleware/errors';
+import { PaymentAccountIdentifier } from 'models/v3/payments-api/common';
+
+export class PaymentsV3NeoController {
```

Comment:
why `Neo`?  and why a different `Controller`?

Reply:
The in theory removes the need for all the other methods, so I created a new Controller which can be updated to replece the other cases.

Reply:
of fair enough, about the name? maybe `PaymentsController`? the repo is public :)

Reply:
Sounds good 👍

Reply:
Done